scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -653,6 +652,38 @@ <h4 class="modal-title">Cancel variant verification</h4>
 {% block scripts %}
   {{ super() }}
   <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/js/bootstrap-select.min.js"></script>
+  <script src="//cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js"></script>
+
+
+
+
+
+  <script>
+    if (document.getElementById('overlapping_snvs_table').rows.length > 5) {
+      $('#overlapping_snvs_table').DataTable({
```

Comment:
If overlapping_snvs_table and genes_table should behave exactly the same you could use a unique javascript scrips maybe?

Reply:
:) smart!